<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KInvestmentDetailsWizardPage</class>
 <widget class="KInvestmentDetailsWizardPage" name="KInvestmentDetailsWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>424</height>
   </rect>
  </property>
  <property name="title">
   <string>Security details</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="textLabel3_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Enter the details below and click &lt;b&gt;Next&lt;/b&gt; to continue entering the online update details.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KMessageWidget" name="m_messageWidget">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="messageType">
      <enum>KMessageWidget::Warning</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout">
     <item row="9" column="1">
      <widget class="KMyMoneyGeneralCombo" name="m_priceMode"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="m_priceModeLabel">
       <property name="text">
        <string>Price entry</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="KComboBox" name="m_tradingMarket">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>AMEX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EUREX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FUND</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NASDAQ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NYSE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XETRA</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="m_pricePrecision">
       <property name="toolTip">
        <string>This determines how many digits after decimal symbol price can have.</string>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="textLabel2_6">
       <property name="text">
        <string>Price precision</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="textLabel1_3">
       <property name="text">
        <string>Trading market</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="m_roundingMethod">
       <property name="toolTip">
        <string>This controls what to do with digit situated after precision digits in amount values.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_investmentSymbol">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel2_2">
       <property name="text">
        <string>Trading symbol</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="textLabel4">
       <property name="text">
        <string>Remainder</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="textLabel1_4">
       <property name="text">
        <string>Trading Currency</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="KMyMoneySecuritySelector" name="m_tradingCurrencyEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="textLabel2_2_2">
       <property name="text">
        <string>Fraction</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="textLabel3">
       <property name="text">
        <string>Full name</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KMyMoneyLineEdit" name="m_investmentName"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="textLabel1_2">
       <property name="text">
        <string>Identification
number</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="KMyMoneyLineEdit" name="m_investmentIdentification"/>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="textLabel1_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>1 /</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AmountEdit" name="m_fraction"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QCheckBox" name="m_createSymbolDuplicate">
       <property name="text">
        <string>Create duplicate of the symbol</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
  <customwidget>
   <class>AmountEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">amountedit.h</header>
  </customwidget>
  <customwidget>
   <class>KInvestmentDetailsWizardPage</class>
   <extends>QWizardPage</extends>
   <header location="global">kinvestmentdetailswizardpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KMyMoneyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>kmymoneylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneySecuritySelector</class>
   <extends>QComboBox</extends>
   <header>kmymoneycurrencyselector.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyGeneralCombo</class>
   <extends>QComboBox</extends>
   <header>kmymoneygeneralcombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_investmentSymbol</tabstop>
  <tabstop>m_investmentName</tabstop>
  <tabstop>m_fraction</tabstop>
  <tabstop>m_tradingMarket</tabstop>
  <tabstop>m_investmentIdentification</tabstop>
  <tabstop>m_tradingCurrencyEdit</tabstop>
  <tabstop>m_pricePrecision</tabstop>
  <tabstop>m_roundingMethod</tabstop>
  <tabstop>m_priceMode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
